DoctorWho/DoctorWho.Db/Repositories/CallsRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace DoctorWho.Db.Repositories
+{
+    public class CallsRepository
```

Comment:
this a very generic name and repo, it has multiple responsibilities. It seems like we can move these methods to the episode repos (Episode, EpisodeCompanion, EpsiodeEnemy repos)